images/federation/Dockerfile

```diff
@@ -0,0 +1,12 @@
+#
+# This is the OpenShift Origin Federation image, used for running the
+# federation apiserver and controller manager components.
+#
+# The standard name for this image is openshift/origin-federation
+#
+FROM openshift/origin-base
```

Comment:
@smarterclayton I'm not sure there's a point in packaging hyperkube, given that this is unlikely to be the method for shipping the federation servers in the future.  What's the simplest acceptable solution to getting this image built for 3.6?